scripts/drone_publish_tag_grafana.sh

```diff
@@ -21,7 +21,7 @@ dagger run --silent go run ./cmd \
   --build-id=${DRONE_BUILD_NUMBER} \
   --grafana-dir=${GRAFANA_DIR} \
   --github-token=${GITHUB_TOKEN} \
-  --version=${DRONE_TAG} \
+  --version=${VERSION} \
```

Comment:
I don't like this. I would prefer a different script for nightly.

Reply:
I just thought of that as well, I am doing that now :)